apps/snmptrapd_sql: Fix build against MySQL 8

See also https://sourceforge.net/p/net-snmp/bugs/2922/.

[bvanassche: changed 'int' into 'char']

Description: apps/snmptrapd_sql: Fix build against MySQL 8
Author: Andreas Hasenack <ahasenack2@users.sourceforge.net>
Origin: upstream, https://github.com/net-snmp/net-snmp/commit/9f4af8c42d
Bug: https://sourceforge.net/p/net-snmp/bugs/2922/
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1814270
Applied-Upstream: v5.8.rc4-180-g9f4af8c42 (drop after v5.8.1.pre1)
Reviewed-by: Rafael David Tinoco <rafaeldtinoco@ubuntu.com>
Last-Update: 2019-12-16

--- a/apps/snmptrapd_sql.c
+++ b/apps/snmptrapd_sql.c
@@ -194,7 +194,7 @@ typedef struct sql_buf_t {
  * static bind structures, plus 2 static buffers to bind to.
  */
 static MYSQL_BIND _tbind[TBIND_MAX], _vbind[VBIND_MAX];
-static my_bool    _no_v3;
+static char       _no_v3;
 
 static void _sql_process_queue(u_int dontcare, void *meeither);
 
